src/test/java/com/epam/qalab/uitests/FilteringAndSearchingTest.java

```diff
@@ -0,0 +1,56 @@
+package com.epam.qalab.uitests;
+
+import com.epam.qalab.BaseTest;
+import com.epam.qalab.pageObject.pages.CategoryPage;
+import com.epam.qalab.pageObject.pages.HomePage;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+import java.util.ArrayList;
+
+public class FilteringAndSearchingTest extends BaseTest {
+
+    @BeforeMethod
+    public CategoryPage openCategoryPage() {
+        HomePage homePage = new HomePage(webDriver);
+        setUpDriver();
+        return homePage.open().choseCategory();
+    }
+
+    @AfterTest
+    public void afterTest() {
+        quit();
+    }
+
+    @Test
+    public void successfulFilteredByFeaturedBrand() {
+
+        var categoryPage = openCategoryPage();
```

Comment:
Maybe I don't really understand the logic. But as I see you call @BeforeMethod here. But if we have annotation @Before on your method, it will be executed automatically.
If you need to get the instance of CategoryPage, for example, you can create a class variable and in this method set the value of your variable

Reply:
Sure! Thank you :)